crates/db_views/src/private_message_view.rs

```diff
@@ -137,3 +147,170 @@ impl JoinView for PrivateMessageView {
     }
   }
 }
+
+#[cfg(test)]
+mod tests {
+  #![allow(clippy::unwrap_used)]
+  #![allow(clippy::indexing_slicing)]
+
+  use crate::private_message_view::PrivateMessageQuery;
+  use lemmy_db_schema::{
+    source::{
+      instance::Instance,
+      local_user::{LocalUser, LocalUserInsertForm},
+      person::{Person, PersonInsertForm},
+      private_message::{PrivateMessage, PrivateMessageInsertForm},
+    },
+    traits::Crud,
+    utils::build_db_pool_for_tests,
+  };
+  use serial_test::serial;
+
+  #[tokio::test]
+  #[serial]
+  async fn test_crud() {
+    let message_content = String::from("");
+    let pool = &build_db_pool_for_tests().await;
+    let pool = &mut pool.into();
+
+    let inserted_instance = Instance::read_or_create(pool, "my_domain.tld".to_string())
+      .await
+      .unwrap();
+
+    let timmy_person_form = PersonInsertForm::builder()
+      .name("timmy_rav".into())
+      .admin(Some(true))
+      .public_key("pubkey".to_string())
+      .instance_id(inserted_instance.id)
+      .build();
+
+    let inserted_timmy_person = Person::create(pool, &timmy_person_form).await.unwrap();
+
+    let timmy_local_user_form = LocalUserInsertForm::builder()
+      .person_id(inserted_timmy_person.id)
+      .password_encrypted("nada".to_string())
+      .build();
+
+    let _inserted_timmy_local_user = LocalUser::create(pool, &timmy_local_user_form)
+      .await
+      .unwrap();
+
+    let sara_person_form = PersonInsertForm::builder()
+      .name("sara_rav".into())
+      .public_key("pubkey".to_string())
+      .instance_id(inserted_instance.id)
+      .build();
+
+    let inserted_sara_person = Person::create(pool, &sara_person_form).await.unwrap();
+
+    let sara_local_user_form = LocalUserInsertForm::builder()
+      .person_id(inserted_sara_person.id)
+      .password_encrypted("nada".to_string())
+      .build();
+
+    let _inserted_sara_local_user = LocalUser::create(pool, &sara_local_user_form)
+      .await
+      .unwrap();
+
+    let jess_person_form = PersonInsertForm::builder()
+      .name("jess_rav".into())
+      .public_key("pubkey".to_string())
+      .instance_id(inserted_instance.id)
+      .build();
+
+    let inserted_jess_person = Person::create(pool, &jess_person_form).await.unwrap();
```

Comment:
I would prefer shorter variable names because they are easier to read, eg simply `jess`.